web/src/test/java/com/microsoft/applicationinsights/web/utils/JettyTestServer.java

```diff
@@ -26,14 +26,41 @@
 import org.eclipse.jetty.servlet.FilterMapping;
 import org.eclipse.jetty.servlet.ServletContextHandler;
 
+import java.io.IOException;
+import java.net.ServerSocket;
+
 /**
  * Created by yonisha on 2/3/2015.
  */
 public class JettyTestServer {
     private Server server;
-    private int Min = 1050;
-    private int Max = 15000;
-    private int portNumber = Min + (int)(Math.random() * ((Max - Min) + 1));
+    private final int Min = 1050;
```

Comment:
@littleaj can you please adhere to camelCase for varable names here. I know you didn't made this variable but since you touched this part of the code better that we can clean up if possible.